src/integrationTest/java/uk/gov/hmcts/darts/common/repository/EventRepositoryTest.java

```diff
@@ -0,0 +1,51 @@
+package uk.gov.hmcts.darts.common.repository;
+
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import uk.gov.hmcts.darts.common.entity.EventEntity;
+import uk.gov.hmcts.darts.testutils.PostgresIntegrationBase;
+import uk.gov.hmcts.darts.testutils.stubs.EventStub;
+import uk.gov.hmcts.darts.testutils.stubs.HearingStub;
+
+import java.util.List;
+import java.util.Map;
+
+class EventRepositoryTest extends PostgresIntegrationBase {
+    @Autowired
+    private EventRepository eventRepository;
+
+    @Autowired
+    private EventStub eventStub;
+
+    @Autowired
+    private HearingStub hearingStub;
+
+    @Test
+    void testEventProcessing() {
+        dartsDatabase.createCase("Bristol", "case1");
+        dartsDatabase.createCase("Bristol", "case2");
+        dartsDatabase.createCase("Bristol", "case3");
+
+        Map<Integer, List<EventEntity>> eventIdMap = eventStub.generateEventIdEventsIncludingZeroEventId(3);
+
+        List<Integer> eventEntityReturned = eventRepository.getCurrentEventIdsToBeProcessed(1);
```

Comment:
can we rename the variable please? the repository is not returning an entity but the event id
```suggestion
        List<Integer> eventIds = eventRepository.getCurrentEventIdsToBeProcessed(1);
```